SPECS/qemu/qemu.spec

```diff
@@ -84,15 +85,18 @@
 %endif
 %define requires_audio_alsa Requires: %{name}-audio-alsa = %{evr}
 %define requires_audio_oss Requires: %{name}-audio-oss = %{evr}
+%if %{with sdl}
 %define requires_audio_sdl Requires: %{name}-audio-sdl = %{evr}
+%else
+%define requires_audio_sdl %{nil}
+%endif
```

Comment:
I'd bundle this with an existing `%if %{with sdl}` section few lines below to scattering of these options.

Reply:
Will do